libraries/AP_Airspeed/AP_Airspeed.cpp

```diff
@@ -887,6 +886,36 @@ bool AP_Airspeed::get_hygrometer(uint8_t i, uint32_t &last_sample_ms, float &tem
 }
 #endif // AP_AIRSPEED_HYGROMETER_ENABLE
 
+#if HAL_GCS_ENABLED
+void AP_Airspeed::send_mavlink_airspeed(mavlink_channel_t chan)
+{
+    for (uint8_t i=0; i<AIRSPEED_MAX_SENSORS; i++) {
+        if (!enabled(i)) {
+            continue;
+        }
+        if (!HAVE_PAYLOAD_SPACE(chan, AIRSPEED)) {
+            return;
+        }
+
+        float temperature_float;
+        int16_t temperature = INT16_MAX;
+        if (get_temperature(i, temperature_float)) {
+            temperature = int16_t(temperature_float * 100);
+        }
+
+        mavlink_airspeed_t msg {
+            airspeed    : get_airspeed(i),
+            raw_press   : get_differential_pressure(i),
+            temperature : temperature,
+            id          : i,
+            flags       : (uint8_t)(healthy(i) | (use(i) << 1U))
```

Comment:
we need a flag for primary

Reply:
should be using an enum from the XML

Reply:
Now using enum. Not added primary, I would like to get this in without waiting for that.